docs/frameworks/dapr.md

```diff
@@ -0,0 +1,112 @@
+
+:::code language="csharp" source="snippets/Dapr/Dapr.AppHost/Program.cs" range="6-16"  highlight="1-7,11":::
+
+Putting everything together, here's an example of a .NET Aspire app host project which includes:
+
+- A backend that declares a Dapr sidecar with specific ports and app ID.
+- A frontend that declares a Dapr sidecar with a specific app ID and default ports.
+
+:::code language="csharp" source="snippets/Dapr/Dapr.AppHost/Program.cs":::
+
+The .NET Aspire dashboard will shows the Dapr sidecar as a resource, with its status and logs.
+
+:::image type="content" source="media/aspire-dashboard-dapr-sidecar-resources.png" lightbox="media/aspire-dashboard-dapr-sidecar-resources.png" alt-text=".NET Aspire dashboard showing Dapr sidecar resources":::
+
+## Adding the Dapr SDK
+
+To use Dapr APIs from .NET Aspire resources you can use the [Dapr SDK for ASP.NET Core](https://www.nuget.org/packages/Dapr.AspNetCore/). The Dapr SDK provides a set of APIs to interact with Dapr sidecars.
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Dapr.AspNetCore
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<PackageReference Include="Dapr.AspNetCore"
+                  Version="[SelectVersion]" />
+```
+
+---
+
+Once installed into an ASP.NET Core project, the SDK can be added to the service builder.
+
+:::code language="csharp" source="snippets/Dapr/Dapr.Web/Program.cs" range="15":::
+
+An instance of `DaprClient` can now be injected into your services to interact with the Dapr sidecar through the Dapr SDK.
+
+:::code language="csharp" source="snippets/Dapr/Dapr.Web/WeatherApiClient.cs" highlight="9-10":::
+
+`InvokeMethodAsync` is a method that sends an HTTP request to the Dapr sidecar. It is a generic method that takes an HTTP verb, the Dapr app ID of the service to call, the method name and a cancellation token. Depending on the HTTP verb it can also take a request body and headers. The generic type parameter is the type of the response body.
+
+The full `Program.cs` file for the frontend project shows the Dapr Client being added to the service builder and the `WeatherApiClient` class that uses the Dapr Client to call the backend service.
+
+:::code language="csharp" source="snippets/Dapr/Dapr.Web/Program.cs" highlight="15,17":::
+
+In, for example, a Blazor project the `WeatherApiClient` class can be injected into a component and used to call the backend service.
+
+:::code language="csharp" source="snippets/Dapr/Dapr.Web/Components/Pages/Weather.razor" highlight="5,47":::
+
+What actually happens when the Dapr SDK is used is that the Dapr sidecar is called over HTTP. The Dapr sidecar then forwards the request to the target service. The target service can be running in the same process as the Dapr sidecar or in a different process. The Dapr sidecar is responsible for service discovery and routing the request to the target service.
```

Comment:
Note: the target service typically isn't running in the same process as the sidecar.  Or, rather, the component related to a service may run in the Dapr sidecar, but that will forward calls (e.g. pub-sub, output bindings, etc.) to some other service.

Reply:
✅